sale_order_general_discount_triple/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Order General Discount Triple",
     "summary": "General discount per sale order with triple",
-    "version": "16.0.2.0.0",
+    "version": "16.0.2.0.1",
```

Comment:
You don't need to update manually version if you don't have migration scripts. OCA bot will do it.